### PR TITLE
Fix some minor issues with the swedish translation

### PR DESCRIPTION
**Pull request by nussun-celestia**

### CODE REVIEW

**Review by levinli303 — APPROVED** [review]

File: po/sv.po [hunk 1]
```diff
@@ -207,12 +207,12 @@ msgstr ""
 #: ../data/data.cpp:46
 #, fuzzy
 msgid "Haumea"
-msgstr "Noumea"
+msgstr "Haumea"
```
Comment (375gnu) [inline po/sv.po:210]: this one is still marked as fuzzy

File: po/sv.po [hunk 2]
```diff
@@ -207,12 +207,12 @@ msgstr ""
...
 
 #: ../data/data.cpp:47
 #, fuzzy
 msgid "Namaka"
-msgstr "Bamako"
+msgstr "Namaka"
```
Comment (375gnu) [inline po/sv.po:215]: the same as above

**Review by 375gnu — CHANGES_REQUESTED** [review]
please rename the commit to have a meaningful name

File: po/sv.po (hunk 1 above)
Comment (nussun-celestia) [reply po/sv.po:210]: Don't know what fuzzy means here

Comment (375gnu) [reply po/sv.po:210]: "Don't use, it's probably wrong"

Comment (nussun-celestia) [reply po/sv.po:210]: Why is it in the code then?

Comment (375gnu) [reply po/sv.po:210]: automatically substitute by gettext program

Comment (nussun-celestia) [reply po/sv.po:210]: should those be reverted then?

Comment (375gnu) [reply po/sv.po:210]: if you fixed a translation you should remove its "fuzzy" line

**Comment by 375gnu** [comment]
as there is activity, i'm closing this pr

**Pull request closed without merging**